webapp/channels/src/actions/post_actions.ts

```diff
@@ -140,6 +141,36 @@ function storeCommentDraft(rootPostId: string, draft: null) {
     };
 }
 
+export function submitReaction(postId: string, action: string, emojiName: string) {
+    return (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState() as GlobalState;
+        const getIsReactionAlreadyAddedToPost = makeGetIsReactionAlreadyAddedToPost();
```

Comment:
I wonder if we can get rid of the `make` step.

Reply:
Shouldn't the `make` step be outside of the `dispatch` function actually?